message LoginRequest {
  required string username = 1;
  required string password = 2;
}

message LoginResponse {
  required bool success = 1;
  required string error = 2;
}

message RequestMessage {
  optional LoginRequest login_request = 1;
}

message ResponseMessage {
  optional LoginResponse login_response = 1;
}
